salesforce/10.9/modules/ROOT/pages/salesforce-connector-upgrade-migrate.adoc

```diff
@@ -0,0 +1,306 @@
+= Upgrade Salesforce Connector to 10.x - Mule 4
+:page-aliases: connectors::salesforce/salesforce-connector-100-upgrade-migrate.adoc
+
+
+Upgrade Anypoint Connector for Salesforce (Salesforce Connector) from version 9.x or 10.x to version 10.9.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|9.x |10.9
+|10.x |10.9
+|===
+
+== Changes
+
+=== 10.9
+
+* Added the Login Request Timeout field and Login Request Time Unit field to all of the connection types
```

Comment:
Should these be bolded since we are referring to the field name?

Reply:
We are no longer bolding filed names. We are using title case now.